packages/theming/atlas/src/web/sass/ui_resources/atlas_ui_resources/buildingblocks/_alert.scss

```diff
@@ -8,11 +8,34 @@
 /* ==========================================================================
    Alerts
 
-//== Design Properties
-//## Helper classes to change the look and feel of the component
+   Default Bootstrap Alert boxes. Provide contextual feedback messages for typical user actions with the handful of available and flexible alert messages
 ========================================================================== */
-.alert-bordered {
+
+.alert {
+    margin-top: 0; // want to align it with padding of a page
+    padding: 15px;
     border: 1px solid;
+    border-radius: 4px;
```

Comment:
Should this be `$border-radius-default`?

Reply:
Yes must have missed one